### PR TITLE
Companies Index/Create/Generate actions & Specs

### PR DESCRIPTION
**Pull request by kushniryb**
Hey
Here's the Rails part of the WT App

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,3 +1,15 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
+  protect_from_forgery with: :null_session
```
Comment (beornborn) [inline app/controllers/application_controller.rb]: protect_from_forgery will not work here at all, we don't send any csrf token

File: app/presenters/company/index/presenter.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+class Company < ApplicationRecord
+  class Index < ::ApplicationOperation
+    class Presenter < ::ApplicationOperation
+      def initialize(params)
+        @companies = params[:companies]
+      end
+
+      def call
+        return companies unless pagination_applied?
```
Comment (beornborn) [inline app/presenters/company/index/presenter.rb]: response format should be always the same. 
for no pagination it should be `{companies: companies}`

File: spec/operations/company/index_spec.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+describe Company::Index do
+  subject { described_class.call }
+
+  describe '.call' do
+    context 'if no records are present in the database' do
+      it { is_expected.to be_empty }
+    end
+
+    context 'if records are present in the database' do
+      let!(:companies)   { create_list(:company, 10) }
```
Comment (beornborn) [inline spec/operations/company/index_spec.rb]: too many spaces

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (kushniryb) [inline app/controllers/application_controller.rb]: Agree, I just thought it's a good additional for an API-style controllers

**Pull request closed without merging**